lib/rpmts.cc

```diff
@@ -275,6 +275,8 @@ static keystore *getKeystore(rpmts ts)
 	    ts->keystore = new keystore_fs();
 	} else if (rstreq(krtype, "rpmdb")) {
 	    ts->keystore = new keystore_rpmdb();
+	} else if (rstreq(krtype, "openpgp")) {
+	    ts->keystore = new keystore_openpgp_cert_d();
```

Comment:
The class name seems a bit verbose to me, how about just `keystore_openpgp`? I suppose this would be the defacto standard openpgp keystore implementation so it doesn't have to be more specific than that.

Reply:
Nevermind the above; I've realized that `openpgp` is way too generic, this whole signature business is about OpenPGP (at least `keystore_rpmdb` is, too) so just keep the name as is, it's good.